### ISSUE TITLE
Word-wrap not preserved when closing and reopening

### ISSUE THREAD

**Issue by JoFr-001** [notepad-plus-plus/notepad-plus-plus#14938]
### Description of the Issue

The "View>Word wrap" selection is not preserved when notepad++ is closed. So this view has to be reactivated every time. This happens on version v8.6.2
### Steps to Reproduce the Issue

1. Select Word wrap
2. Close Notepad++
3. Open Notepad++

### Expected Behavior

Word-wrap will remain activated
### Actual Behavior

Word-wrap is deactivated
### Debug Information

Notepad++ v8.6.2   (32-bit)
Build time : Jan 14 2024 - 02:18:41
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Home (64-bit)
OS Version : 22H2
OS Build : 19045.4170
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.1)
    DSpellCheck (1.4.6)
    mimeTools (3)
    NppConverter (4.5)
    NppExport (0.4)
    NPPJSONViewer (1.21)

**Comment by alankilborn** [comment]
Cannot reproduce on:

Notepad++ v8.6.5   (64-bit)
Build time : Mar 29 2024 - 17:04:43
Path : C:\NotepadPlusPlus\Misc\_NoBackup\Releases\npp.8.6.5\npp.8.6.5.portable.x64\notepad++.exe
Command Line : -titleAdd="8.6.5 64-bit" -multiInst
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 11 Pro (64-bit
OS Version : 23H2
OS Build : 22631.3296
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)

**Comment by JoFr-001** [comment]
I have updated the Notepad++ to  v8.6.5 (64-bit). Problem remains.
I have made the same test on another PC but running Windows 11 Home (64-bit) 23H2. The problem does not exist on this version. I don't want, that you waste time in this small issue. This issue can be closed.
Thanks for looking into it.

- - - 
Notepad++ v8.6.5   (64-bit)
Build time : Mar 29 2024 - 17:04:43
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 11 Home (64-bit
OS Version : 23H2
OS Build : 22631.3296
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
- - -

**Comment by JoFr-001** [comment]
See message above - no furher action required

**Comment by JoFr-001** [comment]
Thanks for you replay - even I've closed it. I have removed all plugins. The problem remains. But again, I'll know how to work with this small issue. Thanks for your reply
--
Notepad++ v8.6.5   (64-bit)
Build time : Mar 29 2024 - 17:04:43
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 10 Home (64-bit
OS Version : 22H2
OS Build : 19045.4170
Current ANSI codepage : 1252
Plugins : none
